Protocol.md

```diff
@@ -318,16 +318,14 @@ Frame Contents
     +-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
     |                           Stream ID                           |
     +-----------+-+-+---------------+-------------------------------+
-    |Frame Type |0|M|      Flags    |
+    |Frame Type |0|0|      Flags    |
     +-----------+-+-+---------------+-------------------------------+
     |                          Error Code                           |
     +---------------------------------------------------------------+
-                        Metadata & Error Data
+                               Error Data
 ```
 
 * __Frame Type__: (6 bits = max value 63) 0x0B
-* __Flags__: (10 bits)
-     * (__M__)etadata: Metadata present
 * __Error Code__: (32 bits = max value 2^31-1 = 2,147,483,647) Type of Error.
      * See list of valid Error Codes below.
 * __Error Data__: includes Payload describing error information. Error Data SHOULD be a UTF-8 encoded string. The string MUST NOT be null terminated.
```

Comment:
If we did do something other than UTF-8, we'd need another MimeType, as we can't assume either Data or Metadata to be appropriate for this. Also, if the MimeType itself is the problem causing the error (bad serialization for example), then is there a fallback to UTF-8?

I would prefer maintaining the simplicity of UTF-8 for error codes so anything can read them even if they don't know the MimeType. 

For the Thrift IDL case, I would just turn each code into a UTF-8 string. For error cases I think the performance of doing so is not a big deal.